app/src/main/java/edu/uc/zhukv/droneradarmap/Weather_Layer/TransparentTileOWM.kt

```diff
@@ -63,7 +74,7 @@ class TransparentTileOWM(private val tileType: String):TileProvider {
      * @return The [URL] of the desired tile image
      */
     private fun getTileUrl(x: Int, y: Int, zoom: Int): URL {
-        val tileUrl = String.format(OWM_TILE_URL, tileType, zoom, x, y)
+        val tileUrl = String.format(OPEN_WEATHER_MAP_TILE_URL, tileType, zoom, x, y)
```

Comment:
Again changed OWM_TITLE_URL to OPEN_WEATHER_MAP_TILE_URL for a more descriptive constant variable name